src/Reflection/SignatureMap/functionMap.php

```diff
@@ -3267,13 +3267,13 @@
 'Gender\Gender::isNick' => ['array', 'name0'=>'string', 'name1'=>'string', 'country='=>'int'],
 'Gender\Gender::similarNames' => ['array', 'name'=>'string', 'country='=>'int'],
 'Generator::__wakeup' => ['void'],
-'Generator::current' => [''],
-'Generator::getReturn' => [''],
-'Generator::key' => [''],
-'Generator::next' => [''],
-'Generator::rewind' => [''],
-'Generator::send' => ['', 'value'=>''],
-'Generator::throw' => ['', 'exception'=>'Exception|Throwable'],
```

Comment:
I noticed these empty strings so I tried to fix them too. Or was there some reason for them to be empty?

Reply:
Empty strings are basically `mixed` but Phan differentiated between them for some minor reason.